CHANGELOG.md

```diff
@@ -28,7 +28,7 @@ Graduation system for users whose FHICT account becomes unavailable. [#287](http
 
 
 ### Fixed
-
+- Fixed inefficient use of entity framework for the fetching-all-projects endpoint.
```

Comment:
```suggestion
- Fixed inefficient use of entity framework for the fetching-all-projects endpoint - [376](https://github.com/DigitalExcellence/dex-backend/issues/376)
```

Small note, we add the link to the issue in the changelog.

Reply:
Fixed